contacts24/models/notes.py

```diff
@@ -1,14 +1,91 @@
-def get_notes():
-    pass
 
+from collections import UserDict
 
-def create_note():
-    pass
+class Note(UserDict):
+    """Цей клас має функціонал для роботи з нотатками"""
+    def add_note(self, client_id, title, text):
+        """Додає нову нотатку до записника."""
+        note_id = len(self.data) + 1  # Генерує унікальний ідентифікатор для нотатки
```

Comment:
@ShulgaSV better to take max index from self.data indeed getting `len`

Test case:
you create new notes
add note 1, 2, 3, 4, 5
and delete note 3
and close and open the application (save/restore file)
and add new note it will be added with index 5, despite you have it already)

Reply:
Done